src/components/ResultViewer.js

```diff
@@ -23,7 +24,12 @@ export class ResultViewer extends React.Component {
   static propTypes = {
     value: PropTypes.string,
     editorTheme: PropTypes.string,
+    ResultsTooltip: PropTypes.any,
   };
+  constructor() {
+    super();
+    this._tooltip = document.createElement('div');
```

Comment:
Also, if `codemirror-graphql` expects an actual DOM element, why can't we just create an actual DOM element when passing in props? From your previous code that's what you'd do to insert tracing information, right?